tests/post_training/test_templates/test_ptq_params.py

```diff
@@ -288,7 +276,8 @@ def test_quantization_points_overflow_fix(self, overflow_fix, affected_target_po
     @pytest.mark.parametrize("validate_scopes", (True, False))
     def test_validate_scope(self, test_params, validate_scopes):
         nncf_graph = test_params["test_model_type_pass"]["nncf_graph"]
-        inference_nncf_graph = transform_to_inference_graph(deepcopy(nncf_graph), [])
+        inference_nncf_graph = deepcopy(nncf_graph)
+        filter_constant_nodes_inplace(inference_nncf_graph)
```

Comment:
This is the place where we will start to mix up every time since we need to understand what really we want to do: `transform_to_inference_graph` using backend-specific entity or just `filter_constant_nodes` using just `NNCFGraph`.
I suggest thinking about current implementation in case of usability.

Reply:
Let's do it in this PR.

Reply:
I reverted transform_to_inference_graph function back